src/plugins/data_source_management/public/components/validation/datasource_form_validation.test.ts

```diff
@@ -49,6 +53,18 @@ describe('DataSourceManagement: Form Validation', () => {
       const result = performDataSourceFormValidation(form, [], '');
       expect(result).toBe(false);
     });
+    test('should fail validation when endpoint path is not empty', () => {
+      form.endpoint = mockDataSourceAttributesWithInvalidEndpoint.endpoint;
+      const result = performDataSourceFormValidation(form, [], '');
+      expect(result).toBe(false);
+    });
+    test('should NOT fail validation when endpoint path is empty', () => {
+      form.auth.type = AuthType.NoAuth;
+      form.title = 'test';
+      form.endpoint = mockDataSourceAttributesWithValidEndpoint.endpoint;
```

Comment:
It seems the `mockDataSourceAttributesWithValidEndpoint.endpoint` is not empty here, shall we update the unit tests description?

Reply:
Thanks for the comment. Working on updating.

Reply:
This test case is not needed. Resolving this comment. Thanks~